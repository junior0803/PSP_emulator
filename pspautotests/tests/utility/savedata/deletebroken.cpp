#include "broken.h"

extern "C" int main(int argc, char **argv) {
	initDisplay();

	for (int i = 1; i <= 3; i++) {
		runBrokenTrials("auto delete", SCE_UTILITY_SAVEDATA_TYPE_AUTODELETE, i);
		runBrokenTrials("delete", SCE_UTILITY_SAVEDATA_TYPE_DELETE, i);
		runBrokenTrials("list delete", SCE_UTILITY_SAVEDATA_TYPE_LISTDELETE, i);
		runBrokenTrials("list all delete", SCE_UTILITY_SAVEDATA_TYPE_LISTALLDELETE, i);
		runBrokenTrials("delete data", SCE_UTILITY_SAVEDATA_TYPE_DELETEDATA, i);
		runBrokenTrials("erase secure", SCE_UTILITY_SAVEDATA_TYPE_ERASESECURE, i);
		runBrokenTrials("erase", SCE_UTILITY_SAVEDATA_TYPE_ERASE, i);
	}

	return 0;
}