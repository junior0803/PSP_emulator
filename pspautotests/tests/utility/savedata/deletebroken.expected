[x] auto delete(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:9): 00000000

[x] delete(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:10): 00000000

[x] list delete(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:6): 00000000
[r] result: 00000000 => 80110347

[x] list all delete(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:7): 00000000
[r] result: 00000000 => 00000001

[x] delete data(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:21): 00000000

[x] erase secure(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:19): 00000000
[r] result: 00000000 => 80110329

[x] erase(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:20): 00000000
[r] result: 00000000 => 80110329

[x] auto delete(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:9): 00000000

[x] delete(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:10): 00000000

[x] list delete(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:6): 00000000
[r] result: 00000000 => 80110347

[x] list all delete(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:7): 00000000
[r] result: 00000000 => 00000001

[x] delete data(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:21): 00000000

[x] erase secure(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:19): 00000000
[r] result: 00000000 => 80110326

[x] erase(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:20): 00000000
[r] result: 00000000 => 80110326

[x] auto delete(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:9): 00000000

[x] delete(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:10): 00000000

[x] list delete(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:6): 00000000
[r] result: 00000000 => 80110347

[x] list all delete(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:7): 00000000
[r] result: 00000000 => 00000001

[x] delete data(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:21): 00000000

[x] erase secure(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:19): 00000000
[r] result: 00000000 => 80110326

[x] erase(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:20): 00000000
[r] result: 00000000 => 80110326
